sdk/textanalytics/ai-text-analytics/src/util.ts

```diff
@@ -16,14 +25,20 @@ export function sortByPreviousIdOrder<T extends IdObject, U extends IdObject>(
   sortedArray: T[],
   unsortedArray: U[]
 ): U[] {
-  if (sortedArray.length !== unsortedArray.length) {
-    throw new Error("Can't use this sort unless you have two arrays with the same length!");
-  }
-
   const unsortedMap = new Map<string, U>();
   for (const item of unsortedArray) {
+    if (isGenericError(item)) {
```

Comment:
maybe update the function comment about this behavior, since this was originally intended to be a general utility function